frontend/src/app/commonComponents/RadioGroup.jsx

```diff
@@ -27,7 +27,7 @@ const RadioGroup = ({
       <li className={classNames} key={button.value}>
         <input
           className={`usa-${type || "radio"}__input`}
-          checked={button.value === selectedRadio || button.checked}
+          checked={button.value === selectedRadio || button.checked || false}
```

Comment:
Without this, the control switches from an uncontrolled one to a controlled one when you click it.